pkg/didcomm/protocol/didexchange/service_test.go

```diff
@@ -122,182 +122,165 @@ func TestSendResponse(t *testing.T) {
 	require.Error(t, prov.SendExchangeResponse(nil, destinationURL))
 }
 
-func TestService_Handle(t *testing.T) {
+// did-exchange flow with role Inviter
+func TestService_Handle_Inviter(t *testing.T) {
 	dbstore, cleanup := store(t)
 	defer cleanup()
 	m := mockProvider{}
 	s := &Service{outboundTransport: m.OutboundTransport(), store: dbstore}
+	thid := randomString()
 
-	// Invitation is sent by Alice
+	// Invitation was previously sent by Alice to Bob.
+	// Bob now sends a did-exchange Request
 	payloadBytes, err := json.Marshal(
-		&Invitation{
-			Type:  "did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/didexchange/1.0/invitation",
-			ID:    "12345678900987654324",
-			Label: "Alice",
-			DID:   "did:sov:QmWbsNYhMrjHiqZDTUTEJs",
-		})
-	require.NoError(t, err)
-
-	msg := dispatcher.DIDCommMsg{Type: "did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/didexchange/1.0/invitation", Payload: payloadBytes} //nolint:lll
-	err = s.Handle(msg)
-	require.NoError(t, err)
-
-	// Invitation accepted and Bob is sending exchange request to Alice
-	payloadBytes, err = json.Marshal(
 		&Request{
 			Type:  "did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/didexchange/1.0/request",
-			ID:    "5369752154652",
+			ID:    thid,
 			Label: "Bob",
 			Connection: &Connection{
 				DID: "B.did@B:A",
 			},
 		})
 	require.NoError(t, err)
-
-	msg = dispatcher.DIDCommMsg{Type: "did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/didexchange/1.0/request", Payload: payloadBytes} //nolint:lll
+	msg := dispatcher.DIDCommMsg{Type: "did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/didexchange/1.0/request", Payload: payloadBytes} //nolint:lll
```

Comment:
In several cases I am seeing a constant used for the DIDComm type, but in a couple cases not?

Reply:
What level of consistency are we looking for? Across the whole test file, within a test function... ?

Reply:
I guess I'll use literals then

Reply:
Fixed

Reply:
I had thought constants were the right way to go :)

Reply:
Ok. I'll revert to constants everywhere except the `TestService_Accept` test cases - I believe those should stay exactly as they are.